frontend/src/components/ScheduledEventsTableCard/ScheduledEventsTableCard.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
```

Comment:
When we moved from TSLint to ESLint, we can import this way `import React from 'react'` ... all other stuff are imported from react via destructuring e.g. `import { FunctionComponent } from 'react'` ... so you don't have to do `React.FunctionComponent`